build/cqs.tsv

```diff
@@ -248,6 +248,8 @@ tec.uom:uom-se	1.0.8	compile
 com.101tec:zkclient	0.10	provided
 com.datastax.cassandra:cassandra-driver-core	3.0.0	provided
 com.datastax.cassandra:cassandra-driver-mapping	3.0.0	provided
+io.confluent:kafka-avro-serializer	5.1.0	provided
```

Comment:
I had this at 5.0.0, but just bumped it to 5.1.0.  License was apache 2.0, but 5.1.0+ is not.  Do I need to leave this at 5.0.0 and allow people to provide 5.1.0?  Or does this dependency need to be removed since we might want to bump kafka in the future without an old confluent potentially breaking the build?

https://twitter.com/nehanarkhede/status/1073594002945929218

Reply:
Actually this one is apache 2:
https://github.com/confluentinc/schema-registry/blob/master/avro-serializer/LICENSE

The 2 below are not:
https://github.com/confluentinc/schema-registry/blob/master/core/LICENSE

As they just broke out the client, I will want to change to that and then we should be good to roll with 5.1.0:
https://github.com/confluentinc/schema-registry/blob/master/client/LICENSE

Reply:
It seems like the confluent license is fairly permissive. @jnh5y do you think it would pass muster with eclipse?